### DISCUSSION TITLE
Github Copilot leaks phone numbers

### DISCUSSION THREAD

**Discussion by ShiSHcat** [community/community#9584] [Copilot Conversations]
![immagine](https://user-images.githubusercontent.com/51242428/147938803-26f8e306-a9fc-4c18-bdb4-63e95fb0c225.png)
This is not some dummy phone number, it really looks like a real mobile phone number.

**Comment by TheDiamondOG** [comment]
It could be a part of code that just looked like a phone number

**Reply by ShiSHcat** [reply]
It's a real and valid phone number though, there is no way that it's dummy.
I also looked up the number and found that it's owned by a telemarketing company.

**Reply by TheDiamondOG** [reply]
Ok that is weird
